docs/user-guide/migrating-to-5.0.0.md

```diff
@@ -194,6 +196,22 @@ When parsing JSX code like `<a>foo</a>`, the default parser will now give the `f
 
 **To address:** If you have written a custom rule that relies on text nodes in JSX elements having the `Literal` type, you should update it to also work with nodes that have the `JSXText` type.
 
+## <a name="spread-operators"></a> When using the default parser, spread operators now have type `SpreadElement`
+
+Previously, when parsing JS code like `const foo = {...data}` with the `experimentalObjectRestSpread` option enabled the default parser would set the `...data` AST node the `ExperimentalSpreadProperty` type.
+
+In ESLint v5, the default parser will now always give the `...data` AST node the `SpreadElement` type, even if the [now deprecated]](#experimental-object-rest-spread) `experimentalObjectRestSpread` option was enabled. This makes the AST compliant with the current ESTree spec.
```

Comment:
Should this say:

<pre>[[now deprecated] `experimentalObjectRestSpread`](#experimental-object-rest-spread)</pre>

Or maybe:

<pre>\[now deprecated\] [`experimentalObjectRestSpread`](#experimental-object-rest-spread)</pre>

Or for that matter, we could use parentheses to make this simpler.

Also: I think instead of "option was enabled", let's use "option is enabled" (to be consistent with present/future tense of this paragraph).

Reply:
Oh, I left a spare `]`. It was:
```md
[now deprecated](#experimental-object-rest-spread) `experimentalObjectRestSpread`
```
As you suggest I will use parentheses instead.